### ISSUE TITLE
Решётка не крепится к lattice

### ISSUE THREAD

**Issue by GodzerBoot** [TauCetiStation/TauCetiClassic#11308]
#### Подробное описание проблемы
При попытке поставить обычную решётку из прутьев поверх lattice решётка будет свободно летать (её можно тягать за собой).  Таким образом барьер вокруг спутника и брига нельзя восстановить в раундстартовом виде.
#### Что должно было произойти
Решётка статично крепится основе из lattice'a
#### Что произошло на самом деле
Решётка свободно летает в космосе
#### Как повторить
Попытаться снять и поставить решётку вокруг спутника ИИ.
#### Дополнительная информация:
Issue reported from  Round ID: 58744 (Tau Ceti Classic (RU))
Testmerges: ```#11137 #11193 #11246 #11250 ```
Reporting client version: 514.1589

**Comment by Naroke** [comment]
Если я правильно понял, то надо просто отвёрткой тыкнуть и всё будет нормально

**Comment by GodzerBoot** [comment]
> Если я правильно понял, то надо просто отвёрткой тыкнуть и всё будет нормально

Нет, это первое, что я попробовал, не работает. Решётка не считает основание за пол к которому можно крепиться.
